recipes/poco/all/conandata.yml

```diff
@@ -2,15 +2,6 @@ sources:
   1.8.1:
     sha256: 43cc469c01d1f799efc51e2bfde6ffdf467b98a8a0901e6b33db86958619b3af
     url: http://github.com/pocoproject/poco/archive/poco-1.8.1-release.tar.gz
-  1.9.0:
-    sha256: 45ec5c759d2ad02bb812745f6cb585aa04af025c921b1bb4bdefa108ce4f1756
-    url: http://github.com/pocoproject/poco/archive/poco-1.9.0-release.tar.gz
-  1.9.1:
-    sha256: 8d76989d8ba5a7080e6a14f9274ff22fda37906fb8991727c2639b26d901198b
-    url: http://github.com/pocoproject/poco/archive/poco-1.9.1-release.tar.gz
-  1.9.2:
-    sha256: c04ee41304645a08aeff8b24321164b101adfc07b3e464fbb9e1856fbd8e4d7b
-    url: http://github.com/pocoproject/poco/archive/poco-1.9.2-release.tar.gz
```

Comment:
Because they are not in `config.yml`?

Reply:
The current error is about shared MT.
I forgot to exclude that combo.